docker/Dockerfile

```diff
@@ -0,0 +1,80 @@
+#
+# Dockerfile for Miaou development
+#
+
+
+
+# Based on docker official ubuntu LTS image
+FROM ubuntu:14.04
+
+
+
+# Define Dockerfile maintainer
+MAINTAINER Oliver THEBAULT <contact@oliver-thebault.com>
+
+
+
+# Install Ubuntu miscellaneous packages
+RUN \
+	sed -i 's/# \(.*multiverse$\)/\1/g' /etc/apt/sources.list && \
+	apt-get update && \
+	apt-get -y upgrade && \
+	apt-get install -y \
+		build-essential \
+		software-properties-common \
+		curl wget unzip xz-utils \
+		git nano vim man htop && \
```

Comment:
`htop` is clearly not needed, and we can use `apt-get install htop` in the running container ourself if we want, so yes I'll remove it

`git` could be used directly inside the container to contribute to Miaou (personnaly, I use it in my host machine but maybe someone else won't want to install it) so I think it's a good idea to keep it (it would be a PITA if a contributor had to `apt-get install git` each time he removes a container).

I always use `nano` on ubuntu, even in the container when I want to check something fast... But OK, we can remove `nano` and `vim`. That was mainly useful while creating the Dockerfile